acloud/resource_cluster.go

```diff
@@ -195,9 +198,22 @@ func getStringAttributeWithLegacyName(d *schema.ResourceData, names ...string) s
 	return defaultValue
 }
 
+func getOrganisation(m interface{}, d *schema.ResourceData) (string, error) {
```

Comment:
```suggestion
func getOrganisation(provider ConfiguredProvider, d *schema.ResourceData) (string, error) {
```
I would pass the `ConfiguredProvider` to this helper method. It should not need to cast to `ConfiguredProvider`.